src/org/omegat/core/team2/RemoteRepositoryFactory.java

```diff
@@ -25,10 +26,12 @@
 
 package org.omegat.core.team2;
 
-import org.omegat.core.team2.impl.FileRepository;
+import java.util.HashMap;
+import java.util.Map;
+
 import org.omegat.core.team2.impl.GITRemoteRepository2;
-import org.omegat.core.team2.impl.HTTPRemoteRepository;
 import org.omegat.core.team2.impl.SVNRemoteRepository2;
```

Comment:
We cannot remove dependency for Git and SVN plugins from OmegaT core because of detection of repository type in Factory class.

```
            if (GITRemoteRepository2.isGitRepository(url)) {
                return "git";
            } else if (SVNRemoteRepository2.isSVNRepository(url)) {
                return "svn";
            }

```

It prevent from making SVN and GIT repository plugins as module.